src/jemalloc.c

```diff
@@ -1252,7 +1252,7 @@ malloc_conf_init(sc_data_t *sc_data, unsigned bin_shard_sizes[SC_NBINS]) {
 			    false)
 			CONF_HANDLE_SIZE_T(opt_lg_extent_max_active_fit,
 			    "lg_extent_max_active_fit", 0,
-			    (sizeof(size_t) << 3), yes, yes, false)
+			    (sizeof(size_t) << 3), no, yes, false)
```

Comment:
I hate these raw "yes"/"no" things a long ways after the macro definition. I wish it were check_min and check_max or something.

Reply:
I find myself constantly looking to figure out which is which too. A quick refactoring task for our new team member maybe ;)